tools/cli/Makefile

```diff
@@ -50,6 +50,12 @@ build:
 	go build -ldflags "$(LINKER_FLAGS)" -o $(DESTINATION) $(SOURCE_FILES)
 
 
+.PHONY: run
```

Comment:
@andreaangiolillo low priority question. Do you have any proven way to run cli for testing with our openapi? 
Any gist/documentation that does run CLI?